ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java

```diff
@@ -1406,4 +1439,114 @@ public static Table tableIfExists(ImportTableDesc tblDesc, Hive db) throws HiveE
     }
   }
 
+  public static class LoadTableStateWrapper {
```

Comment:
Why are both LoadTableStateWrapper and DelayExecUtil needed? They are almost alike

Reply:
DelayExecUtil has been renamed to LoadTableStateWrapper.